### ISSUE TITLE
Expand the `agent` functionality

### ISSUE THREAD

**Issue by LayneHaber** [connext/nxtp#1877]
- [x]  Remove `forceReceiveLocal` function
- [x]  Implement `forceUpdateSlippage` function
- [ ]  Implement `revert`
- [ ]  Ensure `execute` and `reconcile` cannot happen once a transaction has been `reverted`
- [x] Rename `agent` because nobody likes my names :(

**Comment by ArjunBhuptani** [comment]
Must close #1878 before this can be closed

**Comment by LayneHaber** [comment]
Closing in favor of #1878 -- which is blocked by #1735
